src/bin/taskcluster-npm-cache-get.js

```diff
@@ -45,8 +45,9 @@ async function main() {
   let index = new taskcluster.Index();
   let queue = new taskcluster.Queue();
 
-  let pkgText = fs.readFileSync(args.package, 'utf8')
-  let pkgHash = hash(pkgText);
+  let pkgText = fs.readFileSync(args.package, {encoding: 'utf8'});
+  let pkgHash = hash(pkgText.trim());
+  console.log('package hash =', pkgHash);
```

Comment:
I think we want this but lets formalize the logging via a function or something...